test/StdChains.t.sol

```diff
@@ -144,4 +144,17 @@ contract StdChainsTest is Test {
         assertEq(modifiedChain.chainId, 999999999);
         assertEq(modifiedChain.rpcUrl, "https://modified.chain/");
     }
+
+    function testDontUseDefaultRpcUrl() public {
+        // Should error if default RPCs flag is set to false.
+        setUseDefaultRpcUrls(false);
+        vm.expectRevert(
```

Comment:
@mattsse any idea why this test fails? the error messages appear identical, and it seems there's no hidden characters